sql/core/src/main/scala/org/apache/spark/sql/jdbc/PostgresDialect.scala

```diff
@@ -47,7 +47,10 @@ private case class PostgresDialect()
   private val supportedAggregateFunctions = Set("MAX", "MIN", "SUM", "COUNT", "AVG",
     "VAR_POP", "VAR_SAMP", "STDDEV_POP", "STDDEV_SAMP", "COVAR_POP", "COVAR_SAMP", "CORR",
     "REGR_INTERCEPT", "REGR_R2", "REGR_SLOPE", "REGR_SXY")
-  private val supportedFunctions = supportedAggregateFunctions
+  private val supportedDatetimeFunctions = Set("DATE_TRUNC")
+  private val supportedFunctions =
+    supportedAggregateFunctions ++
+      supportedDatetimeFunctions
```

Comment:
Shall we remove supportedDatetimeFunctions by
`private val supportedFunctions = supportedAggregateFunctions ++ Set("DATE_TRUNC")` ?

Reply:
The aggregate functions is very different from others(string functions, datetime functions and so on) and already defined a lot of aggregate functions.
We can define `supportedDatetimeFunctions` in future if need.

Reply:
Done